### DISCUSSION TITLE
Concept: Use student image in user profile

### DISCUSSION THREAD

**Discussion by SethBurkart123** [SethBurkart123/EvenBetterSEQTA#106] [Ideas]
```html
<div class="userInfosvg" style="object-fit: cover; background-image: url('/seqta/student/photo/get?uuid=${info.personUUID}&format=low'); display: inline-flex; align-items: center; justify-content: center; background-size: cover; background-position: center; border-radius: 100%;"></div>
```
Basically if you add this code into where the user icon next to the name is, then you get your school photos (if they do that) inserted right on in. But I'm not sure how I fell about it, although it makes sense. Thoughts?

**Comment by Jaxx7594** [comment]
sounds good, just add a setting to not show it bc some ppl (including me) wont wanna see their terrible school photos lol

**Reply by SethBurkart123** [reply]
yeah, that's what I wasn't sure about... I don't think anyone want's to.

**Reply by MEGA-Dawg68** [reply]
1. omg the photos are so bad.
2. At my school, student seqta doesn't have our school photos only teacher seqta does. unless I am missing something...
